packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -125,11 +129,6 @@ export const addItemToCart = (payload = {}) => {
                 return thunk(...arguments);
             }
         }
-
-        await Promise.all([
```

Comment:
This now only occurs on success -- failure states should have a different effect on the UI (scroll/error display/etc).